platforms/qhaikuplatform/haiku/qhaikuplatformfontdatabase.cpp

```diff
@@ -119,6 +119,15 @@ void QHaikuPlatformFontDatabase::populateFontDatabase()
 			QStringList families = addTTFile(QByteArray(), file);
 		}
 	}
+
+	// Register aliases for generic names
+	registerAliasToFontFamily("Noto Sans", "Sans Serif");
+	registerAliasToFontFamily("Noto Serif", "Serif");
+
+	font_family fixedFontFamily;
+	font_style fixedFontStyle;
+	be_fixed_font->GetFamilyAndStyle(&fixedFontFamily, &fixedFontStyle);
```

Comment:
Why not do this for "Noto Sans" too?

Reply:
I'm not aware of a way to get a default sans-serif font.

Reply:
I suppose that's fair.